src/Nethermind/Nethermind.Dsl.Test/DslRpcModuleTests.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Threading.Tasks;
+using Nethermind.Api;
+using Nethermind.Api.Extensions;
+using Nethermind.Blockchain.Processing;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Extensions;
+using Nethermind.Core.Test.Builders;
+using Nethermind.Int256;
+using Nethermind.Serialization.Json;
+using NSubstitute;
+using NUnit.Framework;
+
+namespace Nethermind.Dsl.Test
+{
+    public class DslRpcModuleTests
+    {
+        private DslPlugin _dslPlugin;
+        private INethermindApi _api;
+        private Dictionary<int, Interpreter> _interpreter;
```

Comment:
hmmm, here is _interpreter and later you use _interpreters,
will those tests even compile?